mempool/cat/reactor.go

```diff
@@ -241,7 +241,7 @@ func (memR *Reactor) ReceiveEnvelope(e p2p.Envelope) {
 		for _, tx := range protoTxs {
 			ntx := types.Tx(tx)
 			key := ntx.Key()
-			schema.WriteMempoolTx(memR.traceClient, string(e.Src.ID()), key[:], schema.Download)
+			schema.WriteMempoolTx(memR.traceClient, string(e.Src.ID()), key[:], len(tx), schema.Download)
```

Comment:
I haven't used the tx hash in the traced data (never been part of the analysis I did), but I assume it might have been utilized for purposes like tracking duplicates. Instead, could we consider extending the `WriteMempoolTX` with a new parameter for tx size? cc: @evan-forbes